fdbclient/SystemData.cpp

```diff
@@ -633,7 +636,8 @@ const KeyRef configKeysPrefix = configKeys.begin;
 
 const KeyRef perpetualStorageWiggleKey(LiteralStringRef("\xff/conf/perpetual_storage_wiggle"));
 const KeyRef perpetualStorageWiggleLocalityKey(LiteralStringRef("\xff/conf/perpetual_storage_wiggle_locality"));
-const KeyRef wigglingStorageServerKey(LiteralStringRef("\xff/storageWigglePID"));
+const KeyRef wigglingStorageServerPrefix(LiteralStringRef("\xff/storageWiggleID/")); // withSuffix /primary or /remote
```

Comment:
Since these vars are all related to the PW feature, they should have the same prefix.

Reply:
Also, this key has changed from ending in `WigglePID to `WiggleID/` between this PR and FDB 7.0.  Are there any upgrade / downgrade implications?

Reply:
I might be wrong, but I think the key change doesn't really impact the PPW behave correctly. In 7.0, the PPW just read the PID to be wiggled under `\xff/storageWigglePID`, when it upgrades to 7.1 `\xff/storageWigglePID` will never be used because now the PPW just read keys under `\xff/storageWiggleID/`. The same thing applies to downgrade to 7.0.

(However, I think it's worth adding upgrade/downgrade StorageEngineMigration tests.

Reply:
The only complication I can think of is
- Upgrade, old key no longer used, new key created
- Downgrade, old key used again but either the PID in it no longer exists or it does exist but it is not the wiggle target and not excluded, but some other process is.
- Upgrade, new key used again but same possibilities as above just with ID instead of PID.

So long as upon startup the value of each of these keys is checked for relevance and ignored if no longer valid / accurate then there should be no issue.  Orphaning the old key isn't too terrible.

Reply:
In fact the invalid value is implicitly handled in `excludeStorageServersForWiggle` [function](https://github.com/apple/foundationdb/pull/6219/files#diff-f3e66a6ee528a2fb4f9f61766269750c368f639ff9e231b00d6c514bbe5e21a6R2924) in both 7.0 and 7.1. If the PID or UID of the wiggle SS is invalid, this function won’t find any SS in the `server_info` (or `pid2server_info`) so it just return `Void` then the storage wiggler will skip to the next SS.